src/report.rs

```diff
@@ -3,7 +3,10 @@
 //! Build a report as clear as possible as to why
 //! dependency solving failed.
 
-use std::fmt;
+use std::{
```

Comment:
I'd prefer if we keep two lines here to keep the style consistent with the rest of the crate.
```rust
use std::fmt;
use std::ops::{Deref, DerefMut};
```

Reply:
Done ✅